hack/local-development/local-garden/configs/authwebhook.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
I think this file was accidentally checked in?!

Reply:
This is still not resolved, please delete this file